storage/maria/ma_check.c

```diff
@@ -5452,28 +5330,26 @@ static int sort_get_next_record(MARIA_SORT_PARAM *sort_param)
       {
 	if (! searching)
 	  _ma_check_print_info(param,
-                               "Found block with wrong recordlength: %lu "
-                               "at %s\n",
-                               block_info.rec_len,
-                               llstr(sort_param->pos,llbuff));
+            "Found block with wrong recordlength: %lu at %lld\n",
```

Comment:
Is this `\n` extraneous?
Also applies to the counterpart in `storage/myisam/mi_check.c`

Reply:
Yes, it is.

https://github.com/MariaDB/server/commit/adefb7cc81404fb70202fd84f614f3d38daa1173#diff-fa3f6010d2b9c0bd15e824385f2f59933a886d13bd44ade44ea679805a65ba03R5431-R5432